tests/unit/type/list.tcl

```diff
@@ -1251,7 +1253,6 @@ foreach {pop} {BLPOP BLMPOP_LEFT} {
             r del blist1{t} blist2{t}
             r set blist2{t} nolist{t}
             bpop_command_two_key $rd $pop blist1{t} blist2{t} 1
-            $rd $pop blist1{t} blist2{t} 1
```

Comment:
why did you delete this?

Reply:
different PR

Reply:
and tag person from blamelog

Reply:
https://github.com/redis/redis/pull/9373/files#r1105474215